app/components/Overlay.tsx

```diff
@@ -0,0 +1,51 @@
+import { Flex, Button, Hide, Show } from "@nycplanning/streetscape";
+import { ReactNode, useState } from "react";
+import { FilterMenu } from "./FilterMenu";
+
+export const Overlay = () => {
+  const [shouldShowFilterMenu, setShouldShowFilterMenu] = useState(false);
+
+  return (
+    <Flex
+      position="relative"
+      padding={{ base: 3, lg: 8 }}
+      height={"100vh"}
+      width={"100vw"}
+      direction={{ base: "column", lg: "row" }}
+      justify={{ base: "flex-end", lg: "flex-start" }}
+      align={{ base: "center", lg: "flex-start" }}
+      pointerEvents={"none"}
+      sx={{
+        "*": {
+          pointerEvents: "auto",
+        },
+      }}
+    >
+      <Show above="lg">
+        <FilterMenu onClose={() => setShouldShowFilterMenu(false)} />
+      </Show>
+      {shouldShowFilterMenu ? (
+        <Hide above="lg">
+          <FilterMenu onClose={() => setShouldShowFilterMenu(false)} />
```

Comment:
I don't love rendering this component twice to get the responsive behavior but it was the only approach I could come up with that didn't require installing a 3rd party hook to get the screen size in pixels as state.